ar_exercise/exercise.rb

```diff
@@ -34,38 +34,90 @@ class Apartment < ActiveRecord::Base
 all_tenants = Tenant.all
 
 # get the first tenant in the DB
+first_tenant = Tenant.first
+
 # get all tenants older than 65
+retirees = Tenant.where("age > '65'")
+# retirees = Tenant.where("age >?" 65)  this maybe works, too.  Or not.
+# retirees = Tenant.where("age > 65" :age)  this maybe works, too, too
+
 # get all apartments whose price is greater than $2300
+luxury = Apartment.where("monthly_rent > '2300'")
+
 # get the apartment with the address "6005 Damien Corners"
+damien_corners = Apartment.find_by(address: "6005 Damien Corners")
+
 # get all tenants in that apartment
+damien_tenants = Tenant.all.where(apartment: damien_corners)
 
 # Use `each` and `puts` to:
 # Display the name and ID # of every tenant
+
+all_tenants.each do |x|
+  puts x[:name] + " " + x[:id].to_s
+end
+
 # Iterate over each apartment, for each apartment, display it's address and rent price
+Apartment.all.each do |x|
+  puts x[:address] + " " + x[:monthly_rent].to_s
+end
+
+
 # Iterate over each apartment, for each apartment, display it's address and all of it's tenants
+# Apartment.all.each do |x|
+#   puts x[:address] + " " + all_tenants(x[:name])
+# end
 
 ################################################
 # CREATING / UPDATING / DELETING
 ################################################
 
 # Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
 
-# Create 3 new apartments, and save them to the DB
-# Create at least 9 new tenants and save them to the DB. (Make sure they belong to an apartment)
+# # Create 3 new apartments, and save them to the DB
+# the_zone = Apartment.create(address: "2214B 15th St", monthly_rent: 700, sqft: 800, num_beds: 3, num_baths: 1)
+# the_loft = Apartment.create(address: "5050 Cuming St", monthly_rent: 3000, sqft: 600, num_beds: 2, num_baths: 1)
+# the_box = Apartment.create(address: "1500 Pennsylvania Ave", monthly_rent: 5000, sqft: 200, num_beds: 1, num_baths: 1)
+#
+# # Create at least 9 new tenants and save them to the DB. (Make sure they belong to an apartment)
+# mike = Tenant.create(name: "Mike Sindt", age: 30, gender: "Male", apartment: the_zone)
+# mark = Tenant.create(name: "Mark Liljehorn", age: 30, gender: "Male", apartment: the_zone)
+# leland = Tenant.create(name: "Leland Jordon", age: 30, gender: "Male", apartment: the_zone)
+# dave = Tenant.create(name: "Dave Flynn", age: 30, gender: "Male", apartment: the_loft)
+# clark = Tenant.create(name: "Clark Wayne", age: 30, gender: "Male", apartment: the_loft)
+# kent = Tenant.create(name: "Kent West", age: 30, gender: "Male", apartment: the_loft)
+# roger = Tenant.create(name: "Roger Stevenson", age: 30, gender: "Male", apartment: the_box)
+# kendra = Tenant.create(name: "Kendra Smythe", age: 30, gender: "Female", apartment: the_box)
+# debbie = Tenant.create(name: "Debbie Dallas", age: 30, gender: "Female", apartment: the_box)
+
```

Comment:
You've hard coded the apartment_id, which generally a bad practice. Fortunately ActiveRecord is super smart and will add the appropriate apartment_id when we create a tenant on an apartment
```rb
the_zone.tenants.create("a bunch of key value pairs, not including apartment_id")
```